va_explorer/va_data_management/management/commands/run_coding_algorithms.py

```diff
@@ -1,15 +1,47 @@
+import argparse
+import pandas as pd
+import time
 from django.core.management.base import BaseCommand
+from django.db.models import F
 from va_explorer.va_data_management.utils.coding import run_coding_algorithms
+from va_explorer.va_data_management.models import VerbalAutopsy, CauseOfDeath
 
 
 class Command(BaseCommand):
 
     help = 'Run cause coding algorithms'
 
+    def add_arguments(self, parser):
+        # whether or not to overwrite existing CODs - if True, will clear all CODs before running
+        parser.add_argument('--overwrite', type=bool, nargs='?', default=False)
+        parser.add_argument('--cod_fname', type=str, nargs='?', default='old_cod_mapping.csv')
+
     def handle(self, *args, **options):
+        ti = time.time()
+        if options['overwrite']:
+            self.clear_and_save_old_cods(**options)
+
+        print('coding all eligible VAs... ')
         results = run_coding_algorithms()
         num_coded = len(results['causes'])
         num_total = len(results['verbal_autopsies'])
         num_issues = len(results['issues'])
-
+        self.stdout.write(f'DONE. Total time: {time.time() - ti} secs')
         self.stdout.write(f'Coded {num_coded} verbal autopsies (out of {num_total}) [{num_issues} issues]')
+
+    def clear_and_save_old_cods(self, *args, **kwargs):
+        
```

Comment:
Don't need *args here. I would also just pass `cod_fname` directly to `clear_and_save_old_cods` since that's the only option that's being used in this function. 

So in `handle` on line 22 you could just do (since you already have the fallback option set in parser):
` self.clear_and_save_old_cods(options['cod_fname'])`